docs/install/harvester-configuration.md

```diff
@@ -278,6 +279,25 @@ os:
     https_proxy: http://myserver
 ```
 
+### `os.force_mbr`
+
+#### Definition
+
+By default, Harvester uses GPT partitioning scheme on both UEFI and BIOS systems.
+However, you could force using MBR on BIOS systems if you encountered compatibility issues.
+
+!!! note
```

Comment:
The whole `note` should be highlighted in a block:
> NOTE:
> Harvester creates an additional partition for storing VM data when GPT partitioning is used.
> If the MBR partitioning is forced, a partition inside the OS will be created for storing VM data.

Reply:
@nunix few questions here:

- Not sure what "highlighted in a block" means? It's already in a "Note" block:
    ![image](https://user-images.githubusercontent.com/5047724/142516831-5537e132-c624-440a-b5a4-4c2b8fcd8833.png)

- Another thing is about the sentence you suggested:
    >  If the MBR partitioning is forced, **a partition inside the OS will be created for storing VM data.**

    What I was trying to explain is: A partition for storing critical OS data will also be used to store VM data, which means if you created too many VMs, it might cause OS to malfunction because there's no disk space for OS to operate. No partitions inside the OS will be created, it's just the one the OS is already using. (Partition table is created before OS is installed)

Reply:
@johnliu55tw yes sorry for the note, Billy also told me it was part of the framework (I was only seeing it on GitHub). The comment can be discarded.

Concerning the partition, thanks a lot, I understand now which partition we are talking about, and here is a better rephrasing:

If the MBR partitioning is forced, the partition containing OS data will be used to also store VM data.

!!! warning
    The space of an MBR partition is limited, and by creating too much VMs, there is an high chance of causing the OS to malfunction due to the lack of disk space.

Reply:
Tips: The CI has added the preview assets automatically, therefore u can preview the sites directly without pulling the PR and build it locally.
![image](https://user-images.githubusercontent.com/4569037/142880113-aba0a591-b812-43af-ba79-61a1282e3b39.png)